saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/README.md

```diff
@@ -0,0 +1,112 @@
+# Spatial Role Labeling
+In paper [1], the task of spatial role labeling is introduced and an annotation scheme proposed that is language-independent and facilitates the application of machine learning techniques. The framework consists of a set of spatial roles. For a basic example, in the following sentence:
+
+
+Give me [the gray book] (trajector) [on] (spatialindicator) [the big table] (landmark).
+
+
+The phrase headed by the token book is referring to a trajector object, the phrase headed by the token table is referring to the role of a landmark and these are related by the spatial expression on denoted as spatial indicator. The spatial indicator (often a preposition) establishes the type of spatial relation.
+
```

Comment:
you can add `The model has a classification setting that classifies candidate spatial relations which are a set of candidate triplets to positive and negative.  However, the model is interesting in that is does a lot of effort for feature engineering which is much easier to do using Saul here. (some thing kine that?)